source/settings/enable-dev-tools.txt

```diff
@@ -0,0 +1,106 @@
+.. _compass-enable-dev-tools:
+
+======================
+Toggle Chrome DevTools
+======================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+You can toggle the ``enableDevTools`` option to control user access to Chrome 
+DevTools on |compass|. 
+
+About This Task
+---------------
+
+By default, :guilabel:`Enable DevTools` is enabled.
+
+If you set :guilabel:`Set Read-Only Mode`, :guilabel:`Enable MongoDB Shell`, 
+:guilabel:`maxTimeMS`, or :guilabel:`Protect Connection String Secrets`, 
+|compass-short| automatically disables the :guilabel:`Enable DevTools` setting.
+
+Procedure
```

Comment:
Likely opting to stick with "Procedure" since it's been used in other task pages on Compass that have been reviewed and approved! My thought process is that procedure probably works better since not all of these examples are exactly enumerated steps.